Update PS Syntax for Get-AzureRmExpressRouteCircuitPeeringConfig

cocallaw opened the pull request:
Invalid PowerShell command syntax, change "-Circuit" to "-ExpressRouteCircuit" for both examples of Get-AzureRmExpressRouteCircuitPeeringConfig, Public and Microsoft Peering

Get-AzureRmExpressRouteCircuitPeeringConfig [-Name ] -ExpressRouteCircuit [-DefaultProfile ] []

https://docs.microsoft.com/en-us/powershell/module/azurerm.network/get-azurermexpressroutecircuitpeeringconfig?view=azurermps-4.4.1&viewFallbackFrom=azurermps-4.4.0

Correct Syntax for examples - 
Get-AzureRmExpressRouteCircuitPeeringConfig -Name "AzurePublicPeering" -ExpressRouteCircuit $ckt
Get-AzureRmExpressRouteCircuitPeeringConfig -Name "MicrosoftPeering" -ExpressRouteCircuit $ckt

azurecla commented:
Hi __@saltug__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess

cocallaw committed:
Update PS Syntax for Get-AzureRmExpressRouteCircuitPeeringConfig

Invalid PowerShell command syntax, change "-Circuit" to "-ExpressRouteCircuit" for both examples of Get-AzureRmExpressRouteCircuitPeeringConfig, Public and Microsoft Peering

Get-AzureRmExpressRouteCircuitPeeringConfig [-Name ] -ExpressRouteCircuit [-DefaultProfile ] []

https://docs.microsoft.com/en-us/powershell/module/azurerm.network/get-azurermexpressroutecircuitpeeringconfig?view=azurermps-4.4.1&viewFallbackFrom=azurermps-4.4.0

Correct Syntax for examples - 
Get-AzureRmExpressRouteCircuitPeeringConfig -Name "AzurePublicPeering" -ExpressRouteCircuit $ckt
Get-AzureRmExpressRouteCircuitPeeringConfig -Name "MicrosoftPeering" -ExpressRouteCircuit $ckt
